code/mob/living/critter/spider.dm

```diff
@@ -306,6 +343,21 @@
 			src.organHolder.drop_and_throw_organ("brain")
 			src.gib(1)
 
+	critter_attack(target)
+		if(ismob(target))
+			var/datum/targetable/critter/spider_bite/bite = src.abilityHolder.getAbility(/datum/targetable/critter/spider_bite)
+			var/datum/targetable/critter/clownspider_kick/kick = src.abilityHolder.getAbility(/datum/targetable/critter/clownspider_kick)
+			if (!kick.disabled && kick.cooldowncheck() && prob(20))
+				kick.handleCast(target)
+			else if(!bite.disabled && bite.cooldowncheck())
+				bite.handleCast(target)
+			else
+				src.set_a_intent(INTENT_HARM)
+				src.hand_attack(target)
+
+	can_critter_attack()
+		return TRUE
```

Comment:
```suggestion
```

Reply:
This override is important because parent